### PR TITLE
Create tuna-sub

### PR DESCRIPTION
**Pull request by dskmh**

### CODE REVIEW

**Comment by LukeSmithxyz** [comment]
Add it too the index too.

File: src/tuna-sub [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Get a better tuna as possible, I like oil-packed or water is OK. Someone you know has a tuna salad recipe.
+
+## Ingredients
+
+- 1 recipe of your favorite tuna salad 
+- Sandwich bun or bread (a good one)
+- Thinly sliced cheese (your favorite)
+- Sliced tomatoes
+- Baby spinich leaves
```
Comment (LukeGoule) [inline src/tuna-sub:11]: *spinach

File: src/tuna-sub [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Tuna Sub
+
+Get a better tuna as possible, I like oil-packed or water is OK. Someone you know has a tuna salad recipe.
```
Comment (LukeGoule) [inline src/tuna-sub:3]: "Use the best tuna you can find" would be better.

**Comment by LukeSmithxyz** [comment]
Put it in the index.

**Pull request merged**